runners/apex/src/main/java/org/apache/beam/runners/apex/translation/operators/ApexGroupByKeyOperator.java

```diff
@@ -212,13 +244,11 @@ private void processElement(WindowedValue<KV<K, V>> windowedValue) throws Except
         windowedValue.getTimestamp(),
         windowedValue.getWindows(),
         windowedValue.getPane());
-
-    KeyedWorkItem<K, V> kwi = KeyedWorkItems.elementsWorkItem(
-            kv.getKey(),
-            Collections.singletonList(updatedWindowedValue));
-
-    context.setElement(kwi, getStateInternalsForKey(kwi.key()));
-    fn.processElement(context);
+    timerInternals.setKey(kv.getKey());
+    ReduceFnRunner<K, V, Iterable<V>, BoundedWindow> reduceFnRunner =
```

Comment:
All other GABW implementations create a new one for every KeyedWorkItem, and I'm wary of changing this in the current PR. I'm not sure ReduceFnRunner is intended to be reused.